integration/patch_test.go

```diff
@@ -189,6 +197,12 @@ func patch(t *testing.T, ref, patchedTag, path string, ignoreErrors bool, report
 		reportPath = "-r=" + path + "/scan.json"
 	}
 
+	if strings.Contains(ref, "oracle") && reportFile {
```

Comment:
There is a fast fail in the tests if Oracle is being patched with a Trivy scan report. Did you want it to fast fail when an end user passes in an Oracle image during normal Copa usage as well?

Reply:
yes, since we will not be able to patch it we can fast fail with a error message to either ignore errors or update all

Reply:
Rather than fast failing, I've added some checks so that if an Oracle image is passed in and ignore errors is false, `log.Info()` says an Oracle image was detected and ignore errors is being set to true. Doing it this way will mean that even if a user doesn't read the docs on Oracle support, Copa still handles the image type properly without needing to exit.

Reply:
@MiahaCybersec I think it would be better to let the user decide and re-patch if they want --ignore-errors set to true? Since we would want them to be aware of the issue/read the docs because if they re-scan the image they would see the vulns